test/binary_to_text_test.cpp

```diff
@@ -255,13 +255,14 @@ INSTANTIATE_TEST_CASE_P(
                 "%1 = OpTypeFloat 16\n%2 = OpConstant %1 0x1.ff4p+16\n",
                 "%1 = OpTypeFloat 16\n%2 = OpConstant %1 -0x1.d2cp-10\n",
                 // 32-bit floats
-                "%1 = OpTypeFloat 32\n%2 = OpConstant %1 -3.275\n",
+                "%1 = OpTypeFloat 32\n%2 = OpConstant %1 -3.2750001\n",
+                "%1 = OpTypeFloat 32\n%2 = OpConstant %1 16777215\n",
                 "%1 = OpTypeFloat 32\n%2 = OpConstant %1 0x1.8p+128\n", // NaN
                 "%1 = OpTypeFloat 32\n%2 = OpConstant %1 -0x1.0002p+128\n", // NaN
                 "%1 = OpTypeFloat 32\n%2 = OpConstant %1 0x1p+128\n", // Inf
                 "%1 = OpTypeFloat 32\n%2 = OpConstant %1 -0x1p+128\n", // -Inf
                 // 64-bit floats
-                "%1 = OpTypeFloat 64\n%2 = OpConstant %1 -3.275\n",
```

Comment:
For tests that care about some float number but not a particular one, I usually go for something that is an integer when multiplied by a power-of-2.  This one should have bee .375  (i.e. 3/8)